pages/api/comments.js

```diff
@@ -0,0 +1,44 @@
+import Cors from 'micro-cors'
+import request from 'request'
+
+import { BLOG_URL, FORUM_URL } from '../../src/consts'
+
+const cors = Cors({
+  allowedMethods: ['GET', 'HEAD'],
+  origin: BLOG_URL
+})
+
+const getCommentCount = (req, res) => {
+  const {
+    query: { url }
+  } = req
+
+  if (!url.startsWith(FORUM_URL)) {
+    res.status(404).json({ error: `URL should starts with '${FORUM_URL}'` })
```

Comment:
Not sure we need to give details as to how to form a correct response. Maybe just return 400 Bad Request.

Reply:
Make sense, will change.

Reply:
Thanks